easy-ui-react/src/CustomerPortalLayout/CustomerPortalLayout.module.scss

```diff
@@ -1,8 +1,8 @@
 @use "../styles/common" as *;
 @use "../styles/unstyled";
 
-.HostedUILayout {
-  @include component-token("hostedui-layout", "max-width", 1369px);
+.CustomerPortalLayout {
```

Comment:
Everything looks good but wondered about the casing of this change, intentional to be upper cased? (Looking on phone so can't see where it's used)